posts/inside-rust/2020-09-04-Const-Evaluatable.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Const evaluatable feature compatability"
```

Comment:
Evaluatable is a pretty strange word, did you mean evaluable?

Reply:
they apparently mean the same thing and we use "evaluatable" inside of the compiler here.

Not sure why this is the case though :thinking:

Reply:
According to typical dictionaries (e.g. Merriam-Webster) it's not a real English word.The only place where it seems to be used is Wolfram Research software.

Reply:
Maybe this should just say 'evaluation' so it's more clear?

Reply:
BTW: 
```suggestion
title: "Const evaluatable feature compatibility"
```